$NetBSD: patch-ag,v 1.2 2012/07/18 21:55:41 marino Exp $

Fix build with png-1.5.
GzFile change from void * in libz 1.2.6
This change required to compile with libz >= 1.2.6

--- src/Util.cpp.orig	2004-05-20 17:42:37.000000000 +0000
+++ src/Util.cpp
@@ -79,7 +79,7 @@ bool utilWritePNGFile(const char *fileNa
     return false;
   }
 
-  if(setjmp(png_ptr->jmpbuf)) {
+  if(setjmp(png_jmpbuf(png_ptr))) {
     png_destroy_write_struct(&png_ptr,NULL);
     fclose(fp);
     return false;
@@ -984,7 +984,7 @@ void utilWriteData(gzFile gzFile, variab
 
 gzFile utilGzOpen(const char *file, const char *mode)
 {
-  utilGzWriteFunc = (int (*)(void *,void * const, unsigned int))gzwrite;
+  utilGzWriteFunc = (int (*)(gzFile, void * const, unsigned int))gzwrite;
   utilGzReadFunc = gzread;
   utilGzCloseFunc = gzclose;
 
